tests/e2e/utils/README.md

```diff
@@ -138,6 +138,10 @@ This package provides support for enabling retries in tests:
 
 ### REST API `withRestApi`
 
+To use this API you need to install [the Basic Auth plugin](https://github.com/WP-API/Basic-Auth),
```

Comment:
Some suggestions for tweaking the wording a bit here:

> Please note: if you're using a non-SSL environment (such as a Docker container) you will need to use Basic Auth in order to authenticate with the API and use the `withRestApi` methods listed below. To do so, you will need to install the [the Basic Auth plugin](https://github.com/WP-API/Basic-Auth). One way this can be accomplished is by adding `wp plugin install https://github.com/WP-API/Basic-Auth/archive/master.zip --activate` to your `initialize.sh` script.

Let me know what you think!